project/NoBigamy.java

```
@@ -0,0 +1,11 @@
+package project;
+
+import project.models.ParsedGED;
+
+public class NoBigamy {
+	public static void check(String path) {
+        ParsedGED gedData = new ParsedGED(path);
+        System.out.println(gedData.getIndividuals().size());
```

Should we also raise an error if bigamy occurs?

---

How does the comparison of individuals and families guarantee that we can determine bigamy?

---

@robertschaedler3 This file/PR doesn't actually contain the logic for NoBigamy, I just included it as a sample to show how the Switch-Case commands setup could work. Implementation of NoBigamy will come later.

---

@aej11a Okay that makes sense. Thanks